src/pkg/cli/compose/loader.go

```diff
@@ -96,7 +97,7 @@ func getDefaultProjectOptions(workingDir string, extraOpts ...cli.ProjectOptions
 		// get compose file path set by COMPOSE_FILE
 		cli.WithConfigFileEnv,
 		// if none was selected, get default compose.yaml file from current dir or parent folder
-		cli.WithDefaultConfigPath,
+		// cli.WithDefaultConfigPath, NO: this ends up picking the "first" when more than one file is found
```

Comment:
Here is the link:
https://github.com/compose-spec/compose-go/blob/9d0d133e13d0955e27520c6317d08822b7c5de5f/cli/options.go#L165

Should we keep it consistent though?